shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -33,13 +35,20 @@
 
     private Map<Integer, SemanticsObject> mObjects;
     private Map<Integer, CustomAccessibilityAction> mCustomAccessibilityActions;
+    // Track whether the corresponding accessibility node has been created yet, and
+    // if an announcement needs to be made.
+    private Map<Integer, Boolean> mLiveRegions;
+    // Track the labels of all text fields so that we can correctly send content changed
+    // events to trigger live region updates.
+    private Map<Integer, String> mTextFields;
     private final FlutterView mOwner;
     private boolean mAccessibilityEnabled = false;
     private SemanticsObject mA11yFocusedObject;
     private SemanticsObject mInputFocusedObject;
     private SemanticsObject mHoveredObject;
     private int previousRouteId = ROOT_NODE_ID;
     private List<Integer> previousRoutes;
+    private String mPackageName;
```

Comment:
What's this used for? It looks like it's only set in the constructor, but never set?

Reply:
I was using it to allow talkback to label images, but we actually need a stable id like the url or asset path also.  Removed for now